### ISSUE TITLE
[Bug]: Italicized :: seems off on Windows

### ISSUE THREAD

**Issue by bebraw** [MonoLisaFont/feedback#273]
### Edition

Basic (Personal)

### What version are you using?

2.015

### What type of weights are you using?

Default (non-customized)

### Operating system

Windows

### Program

JetBrains IDE

### What happened?

Below:

![image-1](https://github.com/MonoLisaFont/feedback/assets/166921/73f29390-044f-4375-bcda-b51fca51a8e8)
![image](https://github.com/MonoLisaFont/feedback/assets/166921/d1bc2e8e-a39a-43ff-a6f2-bfb41427d771)

Test input: `input::placeholder`

**Comment by MarcusSterz** [comment]
Can you describe the problem?

**Comment by bebraw** [comment]
@MarcusSterz See how `::` renders in the screenshots. It's somehow doing `:` and a dot in the middle. For upright, it's rendering correctly.

**Comment by MarcusSterz** [comment]
OK, looks like the path order got messed up. Weird, I don’t recall touching this ligature lately... 
Will look into it.

**Comment by MarcusSterz** [comment]
So it’s a double colon ligature (white space), right? Looks OK in the specimen section.

**Comment by bebraw** [comment]
@MarcusSterz Yeah, that's the weird thing. I wasn't able to reproduce it at the website either (MacOS, though). I wonder if it could be specific to Windows somehow.

**Comment by MarcusSterz** [comment]
I looked into the Glyphs file. There is no indication for a wrong path that would explain this effect.

<img width="1054" alt="Bildschirmfoto 2024-05-15 um 11 44 47" src="https://github.com/MonoLisaFont/feedback/assets/51958720/8e3ede95-06e7-44db-a0f8-b26c74ac70ce">

**Comment by bebraw** [comment]
I think we might have fixed this in 2.015. Here's a render from WebStorm on Windows 11:

<img width="266" alt="Screenshot 2024-05-24 at 10 57 38" src="https://github.com/MonoLisaFont/feedback/assets/166921/63b57973-8a1c-48ca-a2fb-e1e650a4a3f0">

I sent a query to the user to confirm.
